interface/billing/billing_report.php

```diff
@@ -347,303 +340,336 @@ function MarkAsCleared(Type)
    }
  }
 </script>
-<?php include_once("$srcdir/../interface/reports/report.script.php"); ?><!-- Criteria Section common javascript page-->
-<!-- ================================================== -->
+<?php include_once("$srcdir/../interface/reports/report.script.php"); ?>
+<!-- Criteria Section common javascript page-->
 <!-- =============Included for Insurance ajax criteria==== -->
-<!-- ================================================== -->
-<script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-min-1-7-2/index.js"></script>
 <?php include_once("{$GLOBALS['srcdir']}/ajax/payment_ajax_jav.inc.php"); ?>
-<script type="text/javascript" src="../../library/js/common.js?v=<?php echo $v_js_includes; ?>"></script>
 <style>
 #ajax_div_insurance {
-    position: absolute;
-    z-index:10;
-    background-color: #FBFDD0;
-    border: 1px solid #ccc;
-    padding: 10px;
+ position: absolute;
+ z-index: 10;
+ background-color: #FBFDD0;
+ border: 1px solid #ccc;
+ padding: 10px;
+}
+
+button[type="submit"].subbtn-warning {
+ background: #ec971f !important;
+ color: black !important;
+}
+
+button[type="submit"].subbtn-warning:hover {
+ background: #da8104 !important;
+ color: #fff !important;
 }
 </style>
-<script language="javascript" type="text/javascript">
+<script type="text/javascript">
 document.onclick=TakeActionOnHide;
 </script>
-<!-- ================================================== -->
 <!-- =============Included for Insurance ajax criteria==== -->
-<!-- ================================================== -->
 </head>
 <body class="body_top" onLoad="TestExpandCollapse()">
-
-<p style='margin-top:5px;margin-bottom:5px;margin-left:5px'>
-
-<font class='title'><?php echo xlt('Billing Manager') ?></font>
-
-</p>
-
-<form name='the_form' method='post' action='billing_report.php' onsubmit='return top.restoreSession()' style="display:inline">
-
-<script type="text/javascript" src="../../library/dialog.js?v=<?php echo $v_js_includes; ?>"></script>
-<script type="text/javascript" src="../../library/textformat.js?v=<?php echo $v_js_includes; ?>"></script>
-<script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-datetimepicker-2-5-4/build/jquery.datetimepicker.full.min.js"></script>
-<script language='JavaScript'>
+<div class="container">
+    <p style='margin-top: 5px; margin-bottom: 5px; margin-left: 5px; text-align: left;'>
+        <div class='title'><?php echo xlt('Billing Manager') ?></div>
+    </p>
+    <form name='the_form' method='post' action='billing_report.php' onsubmit='return top.restoreSession()' style="display: inline">
+        <?php Header::setupHeader('datetime-picker'); ?>
```

Comment:
That's the way I originally had it and had trouble with bootstrap losing scope. But, worked for me today. Maybe problem came from all my tests with composer.